src/ui/profiling/uiProfileManager.ts

```diff
@@ -265,15 +263,10 @@ export class UiProfileManager implements IDisposable {
     if (this.activeSessions.size === 1) {
       const session: UiProfileSession = this.activeSessions.values().next().value;
       this.statusBarItem.text = session.status
-        ? localize(
-            'profile.status.single',
-            '$(loading~spin) Click to Stop Profiling ({0})',
-            session.status,
-          )
-        : localize('profile.status.default', '$(loading~spin) Click to Stop Profiling');
+        ? l10n.t('$(loading~spin) Click to Stop Profiling ({0})', session.status)
+        : l10n.t('$(loading~spin) Click to Stop Profiling');
```

Comment:
Can you pull out the `$(loading~spin)` and like `'$(loading~spin)' + ternary`? The translators will almost certainly mess that up. This can be a follow up PR